dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1265,17 +1265,20 @@ def create_user(name, url: '/users.json', code: 201, **user_opts)
           password_confirmation: password,
           name: name,
           age: '16',
-          terms_of_service_version: '1'
+          terms_of_service_version: '1',
+          sign_in_count: 2
         }.merge(user_opts)
       },
       code: code
     )
   end
 end
 
-And(/^I create a (young )?student named "([^"]*)"( and go home)?$/) do |young, name, home|
+And(/^I create a (new )?(young )?student named "([^"]*)"( and go home)?$/) do |new_account, young, name, home|
```

Comment:
I'd like to explore other ways of expressing this idea.  "I create a student" and "I create a new student" don't imply a significant difference in behavior to me.  Still brainstorming on this - I'll comment again with some ideas.

Reply:
`student( who has never signed in)?`

Reply:
:+1: Yeah, that would work.  Much more explicit about what we care about in this situation.